GUI/SensorGadget.cs

```diff
@@ -576,6 +576,8 @@ public class SensorGadget : Gadget {
                 case SensorType.Data:
                   format = "{0:F1} GB";
                   break;
+                case SensorType.SmallData:
                  format = "{0:F0} MB";

```

Comment:
Should be on two lines (like data).

Reply:
This are two lines -> Problem here is Line Ending. I have not converted to Unix (LR) because it will set complete file as changed. if possible you could please unify Line Endings after merge to default (Unix LR)